gm4_metallurgy_moneo/data/moneo_shamir/functions/active.mcfunction

```diff
@@ -0,0 +1,27 @@
+#run from main
```

Comment:
These are 14 nbt serialisations in one file

Reply:
If you know a better way to check for the tool material and execute the function, I'd like to know lol

Reply:
@Sheepe Simple. You store the material max when the shamir is applied once. Then you don't need to branch for each item id, just do the operation.

Reply:
Like, store the material maximum inside of the NBT?

Reply:
Or do you mean, give the player a score of the maximum, meaning it's only recalculated when they start holding the tool.

Reply:
Either way, I think I did what you suggested and the amount of checks should be greatly reduced.

Reply:
I actually meant to store it on the tool, but this looks good as well